Source/SocketIOClient/Private/SocketIOClientComponent.cpp

```diff
@@ -406,6 +406,18 @@ void USocketIOClientComponent::Connect(const FString& InAddressAndPort, const FS
 		URLParams.Headers = USIOMessageConvert::JsonObjectToFStringMap(HeadersFJson);
 		URLParams.Path = InPath;
 	}
+	
+	if (Auth != nullptr)
```

Comment:
Missing this check for connect with params if sticking with object flow, otherwise a crash on connect occurs.

Reply:
This should be alleviated with the latest commit 66781294996fd0fd736168154adb6564cc1a91bf